### ISSUE TITLE
[Discussion] Moving official daily builds to the dotnet-core MyGet feed 

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/Home#2937]
Discussion issue for https://github.com/aspnet/Announcements/issues/293

**Comment by roji** [comment]
Just to be sure, should we be moving to the new feed right away? Your announcement says "this will happen over the next few weeks"...

**Comment by natemcmaster** [comment]
The preview1-final packages are currently the latest on dotnet-core. We hope to have an official daily build of preview2 bits up tomorrow on dotnet-core. I thought we'd get it out today, but churn happens so fast that a bunch of things broke. Keep an eye for it in the next few days

**Comment by roji** [comment]
:+1:

**Comment by roji** [comment]
@natemcmaster @ajcvickers @divega can you please clarify which feeds are the right ones for working on both preview2 and preview3? The situation seems to be a bit confusing (is there some docs page?).

dotnet-core
* dotnet-core: latest version of EF Core is 2.1.0-preview2-30478, from March 27th. No preview3.
* aspnetcore-dev: 2.1.0-preview2-30366 (from April 16th), but dependencies seem broken (e.g. depends on System.ComponentModel.Annotations 4.5.0-preview2-26313-01 but 4.5.0-preview2-26326-04 is closest). preview3 seems up to date. 
* aspnetcore-release: 2.1.0-preview2-30518 (from March 30th). No preview3.

So is dotnet-core the place to go for preview2, even though its two weeks behind aspnetcore-dev? For preview3 is aspnetcore-dev the place to go?

**Comment by natemcmaster** [comment]
I recommend using dotnet-core. We're still working on getting our infrastructure to produce preview3 builds on dotnet-core. Turns out building all of .NET Core soup to nuts in a single build system is taking longer than expected. The meantime we've been publishing to aspnetcore-{dev,release} just to unblock our internal daily dev builds and testing, but the feeds are not really mean for external consumption. As you noticed, we stopped mirroring dependencies into aspnetcore-dev, so broken dependencies is expected. dotnet-core is the best place to get a daily build of .NET Core packages which should also have matching installers, Docker images, etc. available.

**Comment by roji** [comment]
@natemcmaster thanks for the answer, I figured as much but I'm still not clear if I should be preparing my preview2 EF Core provider release on dotnet-core, whose latest preview2 package is from March 27th (two whole weeks behind aspnetcore-dev). Can you please confirm which feed is the right one for preview2?

**Comment by natemcmaster** [comment]
Yes, use dotnet-core to prepare your preview2 release. It should be fairly close to what's currently expected to land in preview2.

By the way, if you need matching installers, checkout https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.1

**Comment by natemcmaster** [comment]
By the way, dotnet-core currently has 2.1.0-preview2-30478 which matches this: https://github.com/aspnet/EntityFrameworkCore/tree/c7d8f26c539663788b1390b744dae33c2b657b61

**Comment by roji** [comment]
OK great, thanks!

**Comment by aspnet-hello** [comment]
We periodically close 'discussion' issues that have not been updated in a long period of time.

We apologize if this causes any inconvenience. We ask that if you are still encountering an issue, please log a new issue with updated information and we will investigate.
